apps/cookbook/src/app/extensions/extensions-landing-page.component.html

```diff
@@ -0,0 +1,37 @@
+<article>
+  <h1>Kirby Extensions for Angular</h1>
+  <p>
+    The Kirby Extensions library is a collection of design guidelines and components that build on
+    top of the core Kirby Design library.
+  </p>
+
+  <p>
+    Components in this library are developed and maintained by product teams, with support from
```

Comment:
What is meant by "product teams" - can anyone add components, also people from outside the world of bankdata?

Reply:
Yes, that is the intention. Reworded!